exercises are done

fgsumer opened the pull request:
Hey @segersrobbert ! 

Exercises I started in class are done. 

Have a nice day.

fgsumer committed:
Update index.js

fgsumer committed:
Update index.html

segersrobbert commented:
Good job! But where is the "custom DOM manipulation challenge"?

bitsagarob reviewed (changes requested):
still needs to other homework

fgsumer committed:
Update app.js

fgsumer committed:
Update index.html

fgsumer committed:
Update style.css

fgsumer committed:
Update app.js

Although I followed the same route and they look pretty similar, I don'' know why my first paragraph (name of the authors) are not on the browser.

fgsumer committed:
Add files via upload

fgsumer committed:
Add files via upload

fgsumer committed:
Delete evren_avucunda.jpg
